sqlobject/src/test/java/org/jdbi/v3/sqlobject/customizer/internal/TestTimestamped.java

```diff
@@ -131,7 +131,7 @@ public void shouldUpdateModifiedTimestamp() {
 
         logNext.set("now");
         personDAO.insert(input);
-        Timestamp insert = insertedSqlTimestamp();
+        final Timestamp insert = insertedSqlTimestamp();
```

Comment:
why this change?

Reply:
Due to `VariableDeclarationUsageDistance` check which checks the distance between declaration of variable and its first usage. This check has been in our configuration since 2018.
What strikes me that it only fired now.
We are on the latest version now. Did Checkstyle's behaviour change? There has been some work on this check recently ([release notes](https://checkstyle.sourceforge.io/releasenotes.html)).
I'll increase the `allowedDistance` property from 3 (default) to 10, a fair compromise I think.
The check itself makes sense, having declaration and usage too far apart deteriorates readability.